gen_err.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import random
+import re
+
+action_pattern_map = {
+    'delete(': ("\(", ""),
+    'delete)': ("\)", ""),
+    'delete,': (",", ""),
+    'delete:': (":", ""),
+    'delete{': ("\{", ""),
+    'delete}': ("\}", ""),
+    'delete[': ("\[", ""),
+    'delete]': ("\]", ""),
+    'delete+': ("\+", ""),
+    'delete-': ("-", ""),
+    'delete=': ("=", ""),
+    'delete<': ("<", ""),
+    'delete>': ("\*", ""),
+    'duplicate(': ("\(", "( ("),
+    'duplicate)': ("\)", ") )"),
+    'duplicate,': (",", ", ,"),
+    'duplicate{': ("\{", "{ {"),
+    'duplicate}': ("\}", "} }"),
+    'duplicate[': ("\[", "[ ["),
+    'duplicate]': ("\]", "] ]"),
+    'replace,with;': (",", ";"),
+}
+
+
+action_pattern_map_kw = {
+    'deleteWhile': ("while", ""),
+    'deleteElif': ("elif", ""),
+    'deleteIf': ("if", ""),
+    'deleteIf': ("for", ""),
+    'replaceDefWithDf': ("def", "df"),
+    'replaceElseWithEles': ("else", "eles"),
+    'replaceWhileWithFor': ("while", "for"),
+    'replaceForWithWhile': ("for", "while"),
+    'replaceWhileWithWhil': ("while", "whil"),
+    'replaceClassWithCls': ("class", "cls"),    
+    'replacePandasWithPanda': ("pandas", "panda"),
+    'replace<INDENT>With<DEDENT>': ("<INDENT>", "<DEDENT>"),
+    'replaceNumpyWithNump': ("numpy", "nump"),
+    'replaceRandomWithRandm': ("random", "randm"),
+    'replaceImportWithImport': ("import", "Import"),
+    'replaceIfWithElse': ("if", "else"),
+    'replaceElseWithElif': ("else", "elif"),
+    'replaceElifwithElseif': ("elif", "elseif"),
+}
+
+actions = list(action_pattern_map.keys())
+actions_kw = list(action_pattern_map_kw.keys())
+
+def corrupt_syntax(line):
+
```

Comment:
Is this a good practice to leave empty line at the start of function?

Didn't just mean to follow python code style guide for if else, but the whole code.

Reply:
Empty lines were just for good readability. Will update according to style guide.
Tested the code before committing changes. Code is running as per requirement.

Reply:
>Empty lines were just for good readability.

No worries, the perspective of good readability differs from person to person.

See the usage of blank lines here for reference: https://peps.python.org/pep-0008/#blank-lines